hummingbird/ml/operator_converters/normalizer.py

```
@@ -0,0 +1,31 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import torch
+from onnxconverter_common.registration import register_converter
+
+
+class Normalizer(torch.nn.Module):
+    def __init__(self, norm, device):
+        super(Normalizer, self).__init__()
+        self.norm = norm
+        self.regression = False
+
+    def forward(self, x):
+        if self.norm == "l1":
+            return x / torch.abs(x).sum(1, keepdim=True)
+        elif self.norm == "l2":
+            return x / torch.pow(torch.pow(x, 2).sum(1, keepdim=True), 0.5)
+        elif self.norm == "max":
+            return x / torch.max(torch.abs(x), dim=1, keepdim=True)[0]
+        else:
+            raise RuntimeError("Unsupported norm: {0}".format(self.norm))
```

Please also write a test for `raise RuntimeError("Unsupported norm: {0}".format(self.norm))`.
you can see some example of error test cases [here](https://github.com/microsoft/hummingbird/blob/master/tests/test_sklearn_gbdt_converters.py#L149).